pkgs/tools/networking/rathole/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+, Security
+, CoreServices
```

```suggestion
, CoreServices
```

`CoreServices` already depends on `Security`.